t/parameter.t

```diff
@@ -168,3 +168,36 @@ GET /t
 --- response_body
 match meta: nil
 matched: {}
+
+
+
+=== TEST 6: /name/:name/id/:id
+--- config
+    location /t {
+        content_by_lua_block {
+            local json = require("cjson.safe")
+            local radix = require("resty.radixtree")
+            local rx = radix.new({
+                {
+                    paths = {"/aa/*", "/bb/cc/*", "/dd/ee/index.html"},
+                    uris = {"/aa/*", "/bb/cc/*", "/dd/ee/index.html"},
```

Comment:
I created a new issue: https://github.com/api7/lua-resty-radixtree/issues/43
I think the field `uris` is useless, we should remove it from `resty-radixtree`.